resources/package.rb

```diff
@@ -1,2 +1,6 @@
 actions :install, :uninstall
 default_action :install
+
+attr_accessor :exists
+
+attribute :_setupcmd, :kind_of => String	# not user-visible
```

Comment:
I would like to see the default value for setup command set here rather than in the assignment in the provider.